pkg/antctl/raw/multicluster/delete/member_cluster.go

```diff
@@ -0,0 +1,128 @@
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+	"antrea.io/antrea/pkg/antctl/raw"
+)
+
+type memberClusterOptions struct {
+	namespace  string
+	clusterSet string
+}
+
+var memberClusterOpts *memberClusterOptions
+
+var memberClusterExamples = strings.Trim(`
+Delete a member cluster in a ClusterSet
+$ antctl mc delete member-cluster <MEMBER_CLUSTER> -n <NAMESPACE> --cluster-set <CLUSTER_SET>
+`, "\n")
+
+func (o *memberClusterOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		o.namespace = metav1.NamespaceDefault
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("the ClusterSet cannot be empty")
+	}
+
+	return nil
+}
+
+func NewMemberClusterCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "member-cluster",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Delete a member-cluster in a ClusterSet",
+		Long:    "Delete a member-cluster in a ClusterSet",
+		Example: memberClusterExamples,
+		RunE:    memberClusterRunE,
+	}
+
+	o := &memberClusterOptions{}
+	memberClusterOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of member-cluster")
+	command.Flags().StringVarP(&o.clusterSet, "cluster-set", "", "", "ClusterSet ID of the member-cluster")
+
+	return command
+}
+
+func memberClusterRunE(cmd *cobra.Command, args []string) error {
+	if err := memberClusterOpts.validateAndComplete(); err != nil {
+		return err
+	}
+	if len(args) != 1 {
+		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
+	}
+
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	scheme := k8sruntime.NewScheme()
+	if err = mcsscheme.AddToScheme(scheme); err != nil {
+		return err
+	}
+	if err = antreamcscheme.AddToScheme(scheme); err != nil {
+		return err
+	}
+	if err = k8sscheme.AddToScheme(scheme); err != nil {
+		return err
+	}
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: scheme})
+	if err != nil {
+		return err
+	}
+	memberClusterID := args[0]
+	clusterSet := &multiclusterv1alpha1.ClusterSet{}
+	if err := k8sClient.Get(context.TODO(), types.NamespacedName{Name: memberClusterOpts.clusterSet, Namespace: memberClusterOpts.namespace}, clusterSet); err != nil {
+		return err
+	}
+
+	var memberClusters []multiclusterv1alpha1.MemberCluster
+	for _, m := range clusterSet.Spec.Members {
+		if m.ClusterID != memberClusterID {
+			memberClusters = append(memberClusters, m)
+		}
+	}
+	if len(memberClusters) == len(clusterSet.Spec.Members) {
+		return fmt.Errorf("member-cluster %s not found in ClusterSet %s", memberClusterID, memberClusterOpts.clusterSet)
+	}
+	clusterSet.Spec.Members = memberClusters
+	if err := k8sClient.Update(context.TODO(), clusterSet); err != nil {
+		return err
+	}
+
+	fmt.Fprintf(cmd.OutOrStdout(), "member-cluster %s deleted", memberClusterID)
```

Comment:
s/member-cluster/member cluster/
s/deleted/is deleted/

Reply:
done